### ISSUE TITLE
Would be great to add an option for a team

### ISSUE THREAD

**Issue by destogl** [uesteibar/reviewer-lottery#23]
We have a use case where our open source project is having many reviewers (15-20) sorted in a team.
It would be easier to manage them directly with the team than having to add them manually to each repository with this workflow.

**Comment by intoxicated** [comment]
any progress on this feature?

**Comment by uesteibar** [comment]
nope, haven't had the time to work on it tbh :) if you wanna take on it you're more than welcome!
